src/components/Announcement/Announcement.module.scss

```diff
@@ -0,0 +1,30 @@
+@import 'styles/uswdsDependencies';
+@import 'styles/sfds/sfdsDependencies';
+
+.announcement {
+  position: relative;
+  mix-blend-mode: normal;
+  background-color: $white;
```

Comment:
i think this should be $theme-cosmiclatte-lightest to match the rest of the page?